newIDE/app/src/ProjectManager/PublicGameProperties.js

```diff
@@ -56,6 +69,29 @@ function PublicGameProperties({
           </Trans>
         }
       />
+      {// This view is used for public game properties as well as project properties.
+      // The following properties are not shown in project properties.
+      setPlayableWithKeyboard &&
+        setPlayableWithGamepad &&
+        setPlayableWithMobile && (
+          <React.Fragment>
+            <Checkbox
+              label={<Trans>Playable with keyboards</Trans>}
```

Comment:
`playable with keyboard` without **s** ?

Reply:
Grammatically should it be either "Playable with keyboards" or "Playable with a keyboard"?

Reply:
I feel like we can say "playable with keyboard", but if not I'd got for "playable with a keyboard"

Reply:
Maybe "Keyboard playable"?

Reply:
doesn't sound great, let's go for "Playable with a keyboard" + "Playable with a gamepad" :)